### PR TITLE
expand smart data file limit to 64KB

### PR DESCRIPTION
**Pull request by giggsoff**
Seems, we need increase smart data file limit from current 16KB, we can see 21KB size in our trials.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/pillar/cmd/nodeagent/nodeagent.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ const (
 	minRebootDelay          uint32 = 30
 	maxDomainHaltTime       uint32 = 300
 	domainHaltWaitIncrement uint32 = 5
-	maxReadSize                    = 16384 // From files in /persist
+	maxReadSize                    = 65536 // From files in /persist
```
Comment (eriknordmark) [inline pkg/pillar/cmd/nodeagent/nodeagent.go:59]: Doesn't this affect other reads from /persist? I think we might meed a separate constant for maxSmartCtlSize.

File: pkg/pillar/cmd/zedagent/reportinfo.go [hunk 2]
```diff
@@ -25,14 +24,16 @@ import (
...
 	"github.com/lf-edge/eve/pkg/pillar/zedcloud"
 	"github.com/shirou/gopsutil/host"
 )
 
 var (
-	nilIPInfo = ipinfo.IPInfo{}
-	smartData = types.NewSmartDataWithDefaults()
+	nilIPInfo   = ipinfo.IPInfo{}
+	smartData   = types.NewSmartDataWithDefaults()
+	maxReadSize = 65536 // From files in /persist
```
Comment (eriknordmark) [inline pkg/pillar/cmd/zedagent/reportinfo.go:36]: If we use a different name in nodeagent it makes sense to use the same name here.

**Review by eriknordmark — APPROVED** [review]
Naming nits

File: pkg/pillar/cmd/nodeagent/nodeagent.go (hunk 1 above)
Comment (giggsoff) [reply pkg/pillar/cmd/nodeagent/nodeagent.go]: Good point, fixed.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**